src/cudadecoderbin/Makefile

```diff
@@ -12,6 +12,7 @@ endif
 
 LDFLAGS += $(CUDA_LDFLAGS)
 LDLIBS += $(CUDA_LDLIBS)
+EXTRA_LDLIBS += -lrt
```

Comment:
for a build error like: "batched-wav-nnet3-cuda-online.o: undefined reference to symbol 'clock\_gettime@@GLIBC\_2.2.5'"